src/pages/SignUpResearcher.js

```diff
@@ -52,18 +52,21 @@ function SignUpResearcher() {
   const handleSubmit = async values => {
     try {
       await Db.addResearchData(userId, values);
+
+      setSent(true);
+      notification.open({
+        message: "Success!",
+        description: "Details Updated",
+      });
+      navigate("/");
     } catch (error) {
+      console.error("Error adding researcher details to DB: ", error);
+
       notification.open({
         message: "Error!",
         description: error.message,
       });
     }
-    setSent(true);
-    notification.open({
-      message: "Success!",
-      description: "Details Updated",
-    });
-    navigate('/');
```

Comment:
Similar to previous comment, moved them inside the try block to so that they wont be executed if theres an error